src/pages/organisation.astro

```diff
@@ -0,0 +1,75 @@
+---
+import Cards from "./cards.astro";
+const API_BASE_URL = import.meta.env.API_BASE_URL;
+
+const response = await fetch(`${API_BASE_URL}/projects/1`);
+const allOrganizations = await response.json();
+
+type Projects = {
+  id: number;
+  name: string;
+  description: string;
+  projectUrl: string;
+  programYear: number;
+};
+---
+
+<div class="mainpage">
```

Comment:
Rename the classes. 
And as I understand, this is currently just for providing a proof of concept of the styles right? The rendering for all the organizations would be something that you would work on later?

Reply:
yes bhaiya

Reply:
Okay